textons/filters.cpp

```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <math.h>
 #include <complex>
-//#include "filters.h"
```

Comment:
I don't understand this?

Reply:
This is an artifact from trying to make it work